packages/babel-helpers/src/helpers.js

```diff
@@ -238,30 +238,35 @@ helpers.asyncGeneratorDelegate = () => template.program.ast`
 `;
 
 helpers.asyncToGenerator = () => template.program.ast`
+  function step(gen, resolve, reject, _next, _throw, key, arg) {
```

Comment:
I think it'd be worth giving this a more verbose name if we're splitting it out. Thoughts?

Reply:
Sure.